test/support/TpsData.scala

```diff
@@ -50,6 +49,7 @@ object TpsData {
 
   val id: TpsId = TpsId("session-48c978bb-64b6-4a00-a1f1-51e267d84f91")
   val pciPalSessionId: PcipalSessionId = PcipalSessionId("48c978bb")
+  val paymentItemId: PaymentItemId = PaymentItemId("session-48c978bb-64b6-4a00-a1f1-51e267d84f91")
```

Comment:
Is there any reason why id: TpsId and paymentItemId: PaymentItemId have the same value?

Reply:
Seems odd. They always did (I just renamed the existing val). Want me to make them unique?

Reply:
Yes, please make them unique - so they will clearly represent what would be on production.